XBVC/objects.py

```diff
@@ -65,6 +85,26 @@ def _parse_members(self, msg):
     def __str__(self):
         return self.name
 
+    @property
+    def pascal_name(self):
+        new_name = ""
+        caps = False
+        for c in self.name:
+            if c == '_':
+                caps = True
+                continue
+            new_name += c.upper() if caps else c
+            caps = False
+        return new_name
+
+    @property
+    def camel_name(self):
+        result = ""
+        pname = self.pascal_name
+        result += pname[0].upper()
+        result += pname[1:]
+        return result
```

Comment:
Maybe make these lib functions (in case other emitters need them?) or at least make them a function that both property calls.

Reply:
These are available to all emitters.